program/tests/test_create_mint.rs

```diff
@@ -426,18 +739,28 @@ fn test_successful_token_2022_to_spl_token() {
     let resulting_wrapped_mint_account = &result.resulting_accounts[0].1;
     assert_eq!(resulting_wrapped_mint_account.owner, spl_token::id());
 
-    let wrapped_mint_data = Mint::unpack(&resulting_wrapped_mint_account.data).unwrap();
+    let wrapped_mint_data =
+        PodStateWithExtensions::<PodMint>::unpack(&resulting_wrapped_mint_account.data)
+            .unwrap()
+            .base;
+
     assert_eq!(wrapped_mint_data.decimals, MINT_DECIMALS);
     let expected_mint_authority = get_wrapped_mint_authority(&wrapped_mint_address);
     assert_eq!(
-        wrapped_mint_data.mint_authority.unwrap(),
+        wrapped_mint_data
+            .mint_authority
+            .ok_or(ProgramError::InvalidAccountData)
```

Comment:
nit: is this needed or is it possible to unwrap directly?

Reply:
Unfortunately, [COptionPod](https://github.com/solana-program/token-2022/blob/main/program/src/pod.rs#L162) does not expose a straight unwrap.

Reply:
Ah right, it only has the minimum required interface :sweat_smile: sorry!